Formula/gobby.rb

```diff
@@ -4,7 +4,7 @@ class Gobby < Formula
   url "http://releases.0x539.de/gobby/gobby-0.5.0.tar.gz"
   sha256 "8ceb3598d27cfccdf9c9889b781c4c5c8e1731ca6beb183f5d4555644c06bd98"
   revision 7
-  head "https://github.com/gobby/gobby"
+  head "https://github.com/gobby/gobby.git"
```

Comment:
Please make this a separate PR

Reply:
Will do 👍

Reply:
@SMillerDev I suspect I'll need to also fix [this](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/59678/version=catalina/testReport/junit/brew-test-bot/catalina/audit_gobby___online/) if I am to submit that change, but I'm not sure exactly how to do that.

Can you clarify what adjustment I would need to do to [gobby/0.5.0.patch](https://github.com/Homebrew/formula-patches/blob/master/gobby/0.5.0.patch), and whether that would need to be done before or after the PR in this repo?